manual/src/tutorials/parallelism.etex

```diff
@@ -2,23 +2,23 @@
 %HEVEA\cutname{parallelism.html}
 \label{c:parallelism}
 
-In this chapter, we shall look at the parallel programming facilities in OCaml.
+In this chapter we will look at the parallel programming facilities in OCaml.
```

Comment:
Or just drop the word entirely? I don't mind the tense disagreement with the actual tutorial.
```suggestion
In this chapter we look at the parallel programming facilities in OCaml.
```